### PR TITLE
🐛 removing unsubscribe from rabbitmq + ♻️ comp resource tracking id change

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
 - 🐛 When unsubscribing from the rabbitmq the queue is deleted. What can happen is that during redeployment / or running more replicas firstly the queue is created, but then it is deleted by the previous service (that is shutting down). As this is an import queue that should be always there I removed the unsubscribe from rabbitmq for now. 
 - ♻️ comp resource tracking id change

## Related issue/s

- relates to https://github.com/ITISFoundation/osparc-issues/issues/922

### CODE REVIEW

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py [hunk 1]
```diff
@@ -53,11 +45,12 @@ async def _startup() -> None:
     return _startup
 
 
-def on_app_shutdown(app: FastAPI) -> Callable[[], Awaitable[None]]:
+def on_app_shutdown(
+    app: FastAPI,  # pylint: disable=unused-argument
```
Comment (pcrespov) [inline services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py:49]: This is code-smell (see sonarcloud)

TIP: just ``assert app # nosec `` inside or rename with `_` prefix as `_app` (I prefer the former)

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py [hunk 2]
```diff
@@ -53,11 +45,12 @@ async def _startup() -> None:
...
+) -> Callable[[], Awaitable[None]]:
     async def _stop() -> None:
-        if app.state.resource_tracker_rabbitmq_consumer:
-            await _unsubscribe_from_rabbitmq(app)
-            await app.state.rabbitmq_client.close()
+        # NOTE: We want to have persistent queue, therefore we will not unsubscribe
```
Comment (pcrespov) [inline services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py:52]: Q? is this queue set as durable as well?

**Review by pcrespov — APPROVED** [review]

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py (hunk 2 above)
Comment (matusdrobuliak66) [reply services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py:52]: Yes we have durable=True :+1:

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py (hunk 1 above)
Comment (matusdrobuliak66) [reply services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py]: thanks!

**Review by sanderegg — CHANGES_REQUESTED** [review]
This is not the correct way.
The queue shall be defined as durable pour similar, so that it is not automatically deleted. I can show you how to achieve this tomorrow

**Comment by matusdrobuliak66** [comment]
> This is not the correct way. The queue shall be defined as durable pour similar, so that it is not automatically deleted. I can show you how to achieve this tomorrow

I have changed the resource tracking id (added node id for uniqueness) + For now just unsubscribed from rabbitmq (some tests depend on it, so I will need more time to remove the function totally and will do it some other time)

**Pull request merged**